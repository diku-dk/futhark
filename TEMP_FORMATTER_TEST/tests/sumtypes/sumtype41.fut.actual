b7387654c925fd8b8a44184e157c2de2
