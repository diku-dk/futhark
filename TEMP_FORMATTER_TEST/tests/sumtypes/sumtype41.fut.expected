ff3ab69e328b91da6c22b17e9d779821
